api/test/fixtures/elastic.ts

```diff
@@ -0,0 +1,81 @@
+import { Client, errors } from "@elastic/elasticsearch";
```

Comment:
Is this file being used at all? Since we're now getting stuff from Elasticsearch, I think we can use it to test our API endpoints, as we do in the concepts API.

Reply:
No you're right we can delete it